blocks/hero/hero.css

```diff
@@ -1,7 +1,78 @@
-main .block.hero>div {
+main .block.hero>div, main .block.hero .back-link-base {
   grid-column: 1/-1;
 }
 
+@keyframes link_in {
+  from { transform: translateX(-50%) scaleX(0%); }
+  to { transform: translateX(0%) scaleX(100%); }
+}
+
+@keyframes link_out {
+  from { transform: translateX(0%) scaleX(100%); }
+  to { transform: translateX(-50%) scaleX(0%); }
+}
+
+main .block.hero a.back-link {
+  color: var(--button-tertiary-text-default);
+  font-size: 16px;
+  font-weight: 500;
+  letter-spacing: 0.5px;
```

Comment:
If the links share common style, we can define them globally in styles.css . Similarly the animation pattern seems to repeat everywhere and can be defined globally.

Reply:
I now use the common animation from @alinarublea. Lightly tweaked the common navigation link to change colour on hover and to have the line in the same position as the reference site.